eu.planete_kraus.Tarot.json

```diff
@@ -0,0 +1,42 @@
+{
+    "app-id": "eu.planete_kraus.Tarot",
+    "runtime": "org.freedesktop.Platform",
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "18.08",
+
+    "command": "tarot",
+    "finish-args": [
+	"--socket=x11",
+	"--socket=wayland",
+	"--share=ipc"
+    ],
```

Comment:
Please add something like:

```
"cleanup": [
  "/include"
]
```

…to remove development headers as well.

Reply:
Sorry I had not seen this dialog box.  Should I remove the gir / vapi / typelib?  I did not see a mention of them in the doc (http://docs.flatpak.org/en/latest/manifests.html) but obviously they are development files.

Reply:
OK it works without these.